src/components/itemcontextmenu.js

```diff
@@ -13,14 +13,16 @@ define(['apphost', 'globalize', 'connectionManager', 'itemHelper', 'appRouter',
             if (options.play !== false) {
                 commands.push({
                     name: globalize.translate('Play'),
-                    id: 'resume'
+                    id: 'resume',
+                    icon: "play_arrow"
```

Comment:
can we use consistent quotes please (here and in other places)?

Reply:
done